packages/contracts/contracts/L1/ValidatorManager.sol

```diff
@@ -129,6 +129,12 @@ contract ValidatorManager is ISemver, IValidatorManager {
      */
     mapping(uint256 => uint128) internal _pendingChallengeReward;
 
+    /**
+     * @notice Output index only allowed to submit to TrustedValidator.
+               Challenges for this outputIndex are also restricted.
+     */
+    uint128 public immutable MPT_FIRST_OUTPUT_INDEX;
```

Comment:
It's better to set it uint256, since the type of outputIndex is uint256. We always check the output index with this variable every output submission, so considering the gas usage, uint256 would be better.